module/documents/item.mjs

```diff
@@ -1339,9 +1339,29 @@ export default class Item5e extends SystemDocumentMixin(Item) {
    * @param {DamageRollConfiguration} [config.options]  Additional options passed to the damageRoll function
    * @returns {Promise<DamageRoll[]>}      A Promise which resolves to the created Roll instances, or null if the action
    *                                       cannot be performed.
+   * @deprecated since DnD5e 4.0, targeted for removal in DnD5e 4.4
    */
-  async rollDamage({critical, event=null, spellLevel=null, versatile=false, options={}}={}) {
-    if ( !this.hasDamage ) throw new Error("You may not make a Damage Roll with this Item.");
+  async rollDamage({ spellLevel, ...options }={}) {
+    foundry.utils.logCompatibilityWarning(
+      "The Item5e#rollDamage method has been deprecated and should now be called directly on an activity.",
+      { since: "DnD5e 4.0", until: "DnD5e 4.4" }
+    );
+
+    let item = this;
+    if ( spellLevel ) {
+      if ( this.type === "spell" ) item = item.clone({ "system.item.level": spellLevel });
```

Comment:
```suggestion
      if ( this.type === "spell" ) item = item.clone({ "system.level": spellLevel });
```

Reply:
Seems we have this error also in #3975. Let's make sure to clean that up as well before we merge this.